superset/assets/src/explore/reducers/exploreReducer.js

```diff
@@ -16,19 +16,6 @@ export default function exploreReducer(state = {}, action) {
         isDatasourceMetaLoading: true,
       };
     },
-    [actions.POST_DATASOURCE_SUCCEEDED]() {
```

Comment:
Fixing new warnings, `POST_DATASOURCE_SUCCEEDED` does not exist

Reply:
whoa that's pretty cool that it can resolve that now. wonder if it's babel 7 or a new airbnb preset 🤔